src/pkg/packager/lint/lint_test.go

```diff
@@ -96,23 +94,8 @@ func TestValidateSchema(t *testing.T) {
 		unmarshalledYaml := readAndUnmarshalYaml[types.ZarfPackage](t, badZarfPackage)
 		validator := Validator{typedZarfPackage: unmarshalledYaml}
 		checkForVarInComponentImport(&validator)
-		require.Equal(t, validator.warnings[0], "component.2.import.path will not resolve ZARF_PKG_TMPL_* variables")
-		require.Equal(t, validator.warnings[1], "component.3.import.url will not resolve ZARF_PKG_TMPL_* variables")
+		require.Equal(t, validator.warnings[0], "component.[2].import.path will not resolve ZARF_PKG_TMPL_* variables")
```

Comment:
To be valid `yq` expression, needs a leading `.` and to be called `components`

![Screenshot 2023-11-28 at 4 37 15 PM](https://github.com/defenseunicorns/zarf/assets/50058333/a57fe4bf-528c-4f81-ad78-8280a8213ec4)